src/styles/blocks/card.scss

```diff
@@ -0,0 +1,115 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 198px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  transition: transform 0.3s;
+
+  &__title {
+    margin: 40px 16px 0;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $--text-color;
+  }
+
+  &:hover {
+    transform: scale(1.2);
+
+    & .card__title {
+      color: #34568b;
+    }
+  }
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 0;
+  }
+
+  &__code {
+    margin: 4px 84px 0 16px;
+    line-height: 14px;
+    font-size: 10px;
+    font-weight: 400;
+    color: $--code-color;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 24px 16px 0;
+
+    &-text {
+      margin: 0;
+      line-height: 18px;
+      font-size: 12px;
+      color: $--code-color;
+    }
+
+    &-amount {
+      margin: 0;
+      line-height: 18px;
+      font-weight: 700;
+      font-size: 16px;
+      color: $--text-color;
+    }
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+
+    &-count {
+      margin: 16px 16px 0 0;
+    }
+  }
+
+  &__stars {
+    display: flex;
+    padding: 0;
+    margin: 16px 0 0 16px;
+
+    &-star {
+      padding-right: 4px;
+      background-image: url(../images/star.svg);
+      background-repeat: no-repeat;
+      width: 16px;
+      height: 16px;
+    }
+  }
+
+  &__stars--is-active > :nth-child(-n + 4) {
+    background-image: url(../images/star-active.svg);
```

Comment:
You were supposed to reuse the stars block from the old stars task

For now, it is no easy way to switch stars to show any rating you want
Please fix it